engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/SqlComponentProvider.scala

```diff
@@ -36,15 +37,18 @@ class SqlComponentProvider extends ComponentProvider with LazyLogging {
   override def create(config: Config, dependencies: ProcessObjectDependencies): List[ComponentDefinition] = {
     val parsedConfig = config.rootAs[SqlFileDataSourcesConfig]
 
-    val configs = extractDataSourceConfigFromSqlFileOrThrow(parsedConfig.sqlFilePath)
+    val definition = extractDataSourcesDefinitionFromSqlFileOrThrow(parsedConfig.sqlFilePath)
```

Comment:
As we are getting closer to the first, finished step IMO we should do some polishes:
1. Name of the ComponentProvider may suggest that it is something that communicate over jdbc using SQL. I think that better name would be `FlinkTableComponentProvider` because this SQL file becoming a technical detail how we deliver definition of tables.
2. We will see it when we do an e2e tests, but `tableApi-source-sql`/`tableApi-sink-sql` names look not well on the pallete. Let's rename it to `table` as we have in our user story.
3. I think that we can hide `sql` detail also in sqlsource(factory)/sqlsink(factory). The main purpose of these entities is to read from table and insert into the table. They execute SQL but only in order to achieve this goal.

Reply:
Right, I've done a naming refactor for this whole module :+1: